src/main/java/io/github/thebusybiscuit/slimefun4/utils/HeadTexture.java

```diff
@@ -19,6 +19,7 @@
     TIN_CAN("94da97f080e395b842c4cc82a840823d4dbd8ca688a206853e5783e4bfdc012"),
     MAGNET("aba8ebc4c6a81730947499bf7e1d5e73fed6c1bb2c051e96d35eb16d24610e7"),
     BACKPACK("40cb1e67b512ab2d4bf3d7ace0eaaf61c32cd4681ddc3987ceb326706a33fa"),
+    CONTAINMENT_BACKPACK("a9909a9779b946b9787442fa483af4de4b2f19fd40dc2370f7a9b8f521f21ddc"),
```

Comment:
I think there should be a proper texture for this rather than just a green backpack, could also cause confusion with DyedBackpacks.

Reply:
Never worked with textures, just used it as a "placeholder" tbh